src/main/io/vtx_smartaudio.c

```diff
@@ -702,14 +706,20 @@ bool vtxSmartAudioInit(void)
 #define SA_INITPHASE_WAIT_PITFREQ  2 // SA_FREQ_GETPIT sent and waiting for reply.
 #define SA_INITPHASE_DONE          3
 
-void vtxSAProcess(timeUs_t now)
+void vtxSAProcess(timeUs_t currentTimeUs)
 {
+    UNUSED(currentTimeUs);
+
     static char initPhase = SA_INITPHASE_START;
+    static timeMs_t lastCommandSentMs = 0;
+    static timeMs_t nowMs;
 
     if (smartAudioSerialPort == NULL) {
         return;
     }
 
+    nowMs = millis();
```

Comment:
Any reason not to use `currentTimeUs` here?

Reply:
I just didn't want to do `* 1000` everywhere.

Reply:
Or, you meant `nowMs = currentTimeUs / 1000`?

Reply:
Both would work for me. Calling a function to get the current time in a function that has the current time passed in seems to be counter-intuitive to me.

Reply:
It turned out that `nowMs` must be `millis()`, as `sa_lastTransmissionMs` in `saSendFrame` is set with `millis()`.  `micros() / 1000` != `millis()` after 3000secs.

Reply:
Hehe.